src/sagas/console/index.js

```diff
@@ -268,18 +268,25 @@ function* autoconnect () {
 
   const {
     userId,
+    sessionAgeInSecAtPageLoad,
     websocket,
     defaultVncMode,
     preferredConsole,
     autoconnectOption,
   } = yield select(({ config, options }) => ({
     userId: config.getIn(['user', 'id']),
+    sessionAgeInSecAtPageLoad: config.getIn(['user', 'sessionAgeInSecAtPageLoad'], 0),
     websocket: config.get('websocket'),
     defaultVncMode: config.get('defaultVncMode'),
     preferredConsole: options.getIn(['remoteOptions', 'preferredConsole', 'content'], config.getIn(['defaultUiConsole'])),
     autoconnectOption: toJS(options.getIn(['remoteOptions', 'autoconnect'])),
   }))
 
+  if (sessionAgeInSecAtPageLoad > 15) {
```

Comment:
I suggest making the 15 an item in the config file (`ovirt-web-ui.config` and referenced through `AppConfiguration` in `src/config.js`) to make it easier to change.

Reply:
Based on your followup comment to my review - I also suggest reorganizing the check out to be part of the login saga.  During login, have a function whose purpose is to determine if this is the user's first login.  On a page refresh, it'll run again and should turn off the flag.  Store the result in the config reduce somewhere and just check that flag in the console component.

This would do a few nice things.  First, the react component is just doing a straight lookup to redux, no further logic need apply.  Second, a function is established to do the calculation.

That `isFirstLogin()` function can be as nieve as we want to be able to cover whatever cases we want.  Maybe we stick with sso token age.  Maybe we do something with a cookie with an appropriate scope.  Maybe local storage also with a session type scope.  At the very least, we would have a single place to make that change going forward.